mmtk/rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2021-12-05
+nightly-2022-02-11
```

Comment:
This file does not end with a newline character `'\n'`.  Although mostly benign, it may cause surprising results to some tools, especially scripts.  It is recommended to configure your text editor to always insert the newline character.

Reply:
Done. Thanks.